lib/cli-engine.js

```diff
@@ -754,8 +755,7 @@ CLIEngine.prototype = {
         if (filename && !isAbsolute(filename)) {
             filename = path.resolve(options.cwd, filename);
         }
-        if (filename && ignoredPaths.contains(filename)) {
-
+        if (filename && warnIgnored !== false && ignoredPaths.contains(filename)) {
```

Comment:
yeah, i figured that having a default of true would be preferred to a breaking change.  i'd be happy to update the code throughout this repo to explicitly pass in `true` and have the default be `false` but 3rd party code calling `executeOnText` directly would stop getting these warnings until they added the `true` parameter